sass/mixins/box/box.scss

```diff
@@ -0,0 +1,24 @@
+@import "palette.scss";
+@import "dimensions.scss";
+@import "typography.scss";
+
+@mixin box($type) {
+
+  box-sizing: border-box;
+  display: block;
+  padding: 10px;
+
+  @if $type == 'primary' {
+    background-color: white;
+  }
+  @else if $type == 'secondary' {
+    background-color: $color-150-grey;
+  }
+  @else if $type == 'tertiary' {
+    background-color: $color-300-grey;
+  }
+
+  &[skin=large] {
+    padding: 7px 15px;
```

Comment:
I have `dimensions.scss` but I wanted to keep this generic. So for system wide spacing etc. These styles are specific to the box classes. Maybe we should add some skin partials in to store skin specific variables? Prefix the variables with the component name?